frontend/package.json

```diff
@@ -38,11 +38,11 @@
         "@octokit/rest": "18.12.0",
         "@octokit/types": "6.34.0",
         "@patternfly-labs/react-form-wizard": "^1.7.17",
-        "@patternfly/react-code-editor": "4.52.3",
-        "@patternfly/react-core": "4.211.3",
-        "@patternfly/react-icons": "4.62.3",
-        "@patternfly/react-log-viewer": "4.56.3",
-        "@patternfly/react-table": "4.80.3",
+        "@patternfly/react-code-editor": "4.47.2",
+        "@patternfly/react-core": "4.206.2",
```

Comment:
The bug is avoided if you go back to 4.198.8

https://github.com/stolostron/console/commit/8ce035131574092e6d896786ef5f4f5dd0068314

Reply:
We should probably start bumping minor versions until we find the newest before it breaks.

Reply:
Breaks with version 4.202